sh/setup-migration-script.sh

```diff
@@ -43,14 +43,14 @@ SET collation_connection = 'utf8mb4_unicode_ci';
 " > $MIGRATION_FILE
 
 echo "[migrate] Adding DROP TRIGGERS for $DATABASE."
-mysql -e "SELECT CONCAT('DROP TRIGGER IF EXISTS ', trigger_name, ';') FROM information_schema.triggers WHERE trigger_schema = '$DATABASE';" \
+mysql -u $DB_USER --password=$DB_PASS -e "SELECT CONCAT('DROP TRIGGER IF EXISTS ', trigger_name, ';') FROM information_schema.triggers WHERE trigger_schema = '$DATABASE';" \
```

Comment:
Thanks, this is helpful and explains a lot. 

It's clear to me now.  Why does it work on my machine?  Because I have a "superuser" in my ~/.my.cnf file that has access to all these databases.  So even though this doesn't have a user passed in, it looks up the user from my environmental file.  Whoops!  Thanks for catching this.